README.md

```diff
@@ -115,7 +115,8 @@ you can update your code and run `fn deploy myapp` again.
 
 ## To Learn More
 
-* Visit our Functions [Tutorial Series](examples/tutorial/)
+* Try out a [Tutorial](../tutorials/README.md) 
+* Visit our Functions [Quickstart Series](examples/tutorial/)
```

Comment:
Quickstart is already in the main README, should call this something else. Language hello world examples perhaps?

Reply:
Maybe `Functions development quickstart series`?